Gateway/Response/CheckoutDonationsDetailsHandler.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Gateway\Response;
+
+use Magento\Payment\Gateway\Response\HandlerInterface;
+
+class CheckoutDonationsDetailsHandler implements HandlerInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function handle(array $handlingSubject, array $response)
+    {
+        // @todo: PW-5424 Implement handle() method.
```

Comment:
this class turned out to be unnecessary in the implementation of #1218 
it can be deleted and removed from the di.xml